egs/material/s5/cmd.sh

```diff
@@ -0,0 +1,14 @@
+# you can change cmd.sh depending on what type of queue you are using.
+# If you have no queueing system and want to run on a local machine, you
+# can change all instances 'queue.pl' to run.pl (but be careful and run
+# commands one by one: most recipes will exhaust the memory on your
+# machine).  queue.pl works with GridEngine (qsub).  slurm.pl works
+# with slurm.  Different queues are configured differently, with different
+# queue names and different ways of specifying things like memory;
```

Comment:
I notice there is no data in the RESULTS file (or the README?    Perhaps there should be?

Reply:
I will add the commands needed to run the scripts in the README file.
I will add tables of results from supervised and semi-supervised training in the RESULTS file.

Reply:
Done.